pkg/kubelet/qos/policy.go

```diff
@@ -63,11 +60,6 @@ func GetContainerOOMScoreAdjust(pod *v1.Pod, container *v1.Container, memoryCapa
 	// targets for OOM kills.
 	// Note that this is a heuristic, it won't work if a container has many small processes.
 	memoryRequest := container.Resources.Requests.Memory().Value()
-	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
```

Comment:
Could you add a comment to the function stating that we calculate the oom score based on the allocated resources (as opposed to the desired resources in the spec)?

Reply:
Yeah, I started making the type change out of curiosity, and it ballooned into a big change very quickly. I might still try to do it after all the other beta changes go in.